jena-arq/src/main/java/org/apache/jena/riot/lang/rdfxml/rrx/ParserRDFXML_SAX.java

```diff
@@ -1042,15 +1040,21 @@ private String xmlLang(Attributes attributes, Position position) {
         return langStr;
     }
 
-
     private ObjectParseType objectParseType(String parseTypeStr, Position position) {
         if ( parseTypeStr == null )
             return ObjectParseType.Plain;
         try {
             String parseTypeName = parseTypeStr;
-            if ( parseTypeName.equals("literal") ) {
-                Log.warn(SysRIOT.getLogger(), "Encountered rdf:parseType='literal'. Treated as rdf:parseType='literal'");
-                parseTypeName = "Literal";
+            switch(parseTypeName) {
+                case "literal" -> {
+                    RDFXMLparseWarning("Encountered rdf:parseType='literal'. Treated as rdf:parseType='literal'", position);
+                    parseTypeName = "Literal";
+                }
+                // CIM (Common Information Model) - see github issue 2473
+                case "Statements" -> {
+                    RDFXMLparseWarning("Encountered rdf:parseType='Statements'. Treated as rdf:parseType='literal'", position);
+                    parseTypeName = "Literal";
```

Comment:
Same question as below... I think it's confusing that `rdf:parseType='literal'` appears so many times in the error message. I expected at least one more option/value.

Reply:
I think this is incorrect. The original `if (parseTypeName.equals("literal")) {` should only issue a warning if someone accidentally wrote 'Literal' in lowercase. Therefore, if we extend the same logic to 'Statements', the case should be written in lowercase as 'statements', and the warning should be: 'Encountered rdf:parseType="statements". Treated as rdf:parseType="Literal."